eva/expression/comparison_expression.py

```diff
@@ -77,3 +77,6 @@ def __eq__(self, other):
             return False
         return (is_subtree_equal
                 and self.etype == other.etype)
+
+    def __hash__(self) -> int:
+        return super().__hash__()
```

Comment:
Is this right?

Reply:
Abstract Expression will take care of those

Reply:
Got. Thanks.